conf/config.neon

```diff
@@ -1822,6 +1825,7 @@ services:
 		arguments:
 			parser: @currentPhpVersionPhpParser
 			lexer: @currentPhpVersionLexer
+			enableIgnoreErrorsWithinPhpDocs: %featureToggles.bleedingEdge%
```

Comment:
`phpDocParserIncludeLines` would make more sense here

Reply:
I dont think so, those two are not really bounded. This is just fixed behaviour of ignore-error. Unrelated to phpDocParser setup.

We can name it `%featureToggles.ignoreErrorsWithinPhpDocs%` if you want